README.md

```diff
@@ -165,6 +165,19 @@ git diff step-?..step-?
   - CSS transition animations.
   - CSS keyframe animations.
   - JavaScript-based animations.
+  
+### step-15 _Accessibility (a11y)_
+
+- Add ngAria module.
```

Comment:
Didn't you end up removing it in the end? (Although it does seem strange to have a chapter about ARIA that doesn't mention or need `ngAria` :smiley:)